### PR TITLE
Add OpsGenie to README.md and reformat the list.

### PR DESCRIPTION
**Pull request by juliusv**

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7

File: README.md [hunk 1]
```diff
@@ -15,14 +15,16 @@ following aspects:
...
+* sending alert notifications via external services:
+  * email
+  * generic web hook
+  * [PagerDuty](http://www.pagerduty.com/)
+  * [HipChat](http://www.hipchat.com/)
+  * [Slack](http://www.slack.com/)
+  * [Pushover](https://www.pushover.net/)
+  * [Flowdock](https://www.flowdock.com/)
+  * [Amazon SNS](http://aws.amazon.com/sns/)
+  * [OpsGenie](https://www.opsgenie.com/)
```
Comment (beorn7) [inline README.md]: Perhaps sort lexicographically but keep email and web hook on top?

Comment (juliusv) [inline README.md]: Done.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**